data/net/minecraft/world/level/biome/FeatureSorter.mapping

```diff
@@ -1,4 +1,16 @@
 CLASS net/minecraft/world/level/biome/FeatureSorter
+	METHOD buildFeaturesPerStep (Ljava/util/List;Ljava/util/function/Function;Z)Ljava/util/List;
+		ARG 0 featureSetSources
+		ARG 1 toFeatueSetFunction
+		ARG 2 notRecursiveFlag
+	METHOD lambda$buildFeaturesPerStep$0 (Lorg/apache/commons/lang3/mutable/MutableInt;Ljava/lang/Object;)I
+		ARG 1 ignored
+	METHOD lambda$buildFeaturesPerStep$1 (Ljava/util/Comparator;Lnet/minecraft/world/level/biome/FeatureSorter$1FeatureData;)Ljava/util/Set;
+		ARG 1 ignored
```

Comment:
```suggestion
		ARG 1 key
	METHOD lambda$buildFeaturesPerStep$1 (Ljava/util/Comparator;Lnet/minecraft/world/level/biome/FeatureSorter$1FeatureData;)Ljava/util/Set;
		ARG 1 key
```

Same as above, I think this reads better given it's `.computeIfAbsent(key -> new thing)`